[2022-11-30T01:48:14.872+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v03.delete_from_table scheduled__2022-11-28T00:00:00+00:00 [queued]>
[2022-11-30T01:48:14.885+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v03.delete_from_table scheduled__2022-11-28T00:00:00+00:00 [queued]>
[2022-11-30T01:48:14.885+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-30T01:48:14.885+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 6
[2022-11-30T01:48:14.885+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-30T01:48:14.908+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): delete_from_table> on 2022-11-28 00:00:00+00:00
[2022-11-30T01:48:14.915+0000] {standard_task_runner.py:55} INFO - Started process 2185 to run task
[2022-11-30T01:48:14.920+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'delete_from_table', 'scheduled__2022-11-28T00:00:00+00:00', '--job-id', '184', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmp92p8q9bd']
[2022-11-30T01:48:14.922+0000] {standard_task_runner.py:83} INFO - Job 184: Subtask delete_from_table
[2022-11-30T01:48:15.075+0000] {task_command.py:376} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.delete_from_table scheduled__2022-11-28T00:00:00+00:00 [running]> on host 2c9b3e546def
[2022-11-30T01:48:15.374+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Bruno Lima
AIRFLOW_CTX_DAG_ID=dag_with_postgres_operator_v03
AIRFLOW_CTX_TASK_ID=delete_from_table
AIRFLOW_CTX_EXECUTION_DATE=2022-11-28T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-28T00:00:00+00:00
[2022-11-30T01:48:15.390+0000] {base.py:71} INFO - Using connection ID 'postgres_localhost' for task execution.
[2022-11-30T01:48:15.395+0000] {sql.py:315} INFO - Running statement: 
            delete from dag_runs where dt = '2022-11-28' and dag_id 'dag_with_postgres_operator_v03';
        , parameters: None
[2022-11-30T01:48:15.396+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/postgres/operators/postgres.py", line 94, in execute
    self.hook.run(self.sql, self.autocommit, parameters=self.parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 295, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 320, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.UndefinedObject: type "dag_id" does not exist
LINE 2: ... delete from dag_runs where dt = '2022-11-28' and dag_id 'da...
                                                             ^

[2022-11-30T01:48:15.434+0000] {taskinstance.py:1406} INFO - Marking task as UP_FOR_RETRY. dag_id=dag_with_postgres_operator_v03, task_id=delete_from_table, execution_date=20221128T000000, start_date=20221130T014814, end_date=20221130T014815
[2022-11-30T01:48:15.466+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 184 for task delete_from_table (type "dag_id" does not exist
LINE 2: ... delete from dag_runs where dt = '2022-11-28' and dag_id 'da...
                                                             ^
; 2185)
[2022-11-30T01:48:15.507+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2022-11-30T01:48:15.608+0000] {taskinstance.py:2623} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-11-30T01:49:16.322+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v03.delete_from_table scheduled__2022-11-28T00:00:00+00:00 [queued]>
[2022-11-30T01:49:16.340+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v03.delete_from_table scheduled__2022-11-28T00:00:00+00:00 [queued]>
[2022-11-30T01:49:16.341+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-30T01:49:16.341+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 6
[2022-11-30T01:49:16.341+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-30T01:49:16.428+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): delete_from_table> on 2022-11-28 00:00:00+00:00
[2022-11-30T01:49:16.432+0000] {standard_task_runner.py:55} INFO - Started process 2318 to run task
[2022-11-30T01:49:16.435+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'delete_from_table', 'scheduled__2022-11-28T00:00:00+00:00', '--job-id', '196', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpk4v9a6jt']
[2022-11-30T01:49:16.436+0000] {standard_task_runner.py:83} INFO - Job 196: Subtask delete_from_table
[2022-11-30T01:49:16.640+0000] {task_command.py:376} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.delete_from_table scheduled__2022-11-28T00:00:00+00:00 [running]> on host 2c9b3e546def
[2022-11-30T01:49:16.755+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Bruno Lima
AIRFLOW_CTX_DAG_ID=dag_with_postgres_operator_v03
AIRFLOW_CTX_TASK_ID=delete_from_table
AIRFLOW_CTX_EXECUTION_DATE=2022-11-28T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-28T00:00:00+00:00
[2022-11-30T01:49:16.772+0000] {base.py:71} INFO - Using connection ID 'postgres_localhost' for task execution.
[2022-11-30T01:49:16.786+0000] {sql.py:315} INFO - Running statement: 
            delete from dag_runs where dt = '2022-11-28' and dag_id = 'dag_with_postgres_operator_v03';
        , parameters: None
[2022-11-30T01:49:16.791+0000] {sql.py:324} INFO - Rows affected: 0
[2022-11-30T01:49:16.810+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v03, task_id=delete_from_table, execution_date=20221128T000000, start_date=20221130T014916, end_date=20221130T014916
[2022-11-30T01:49:16.882+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-11-30T01:49:16.932+0000] {taskinstance.py:2623} INFO - 1 downstream tasks scheduled from follow-on schedule check
